2024-RIKEN-AWS/JupyterNotebook/tutorial/notebook/dyad_dlio.ipynb

```diff
@@ -0,0 +1,345 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "dd3e912b-3428-4bc7-88bd-97686406b75a",
```

Comment:
What about the original dyad notebook (see it here https://github.com/flux-framework/Tutorials/tree/4adc40d7519d592b467eac23c04f421383089874/2024-RIKEN-AWS/JupyterNotebook/tutorial/notebook) is this supplement to that or intended to replace it? A

Reply:
The `dyad_dlio.ipynb` is going to replace the existing `dyad.ipynb`. In fact, when I'm done editing it, I'm going to rename `dyad_dlio.ipynb` to `dyad.ipynb`. I'm just keeping the original around until I've finished the text in the new notebook.

Reply:
ok! Ping me when you are all done and ready for a review. 

Actually that's what draft is for right?

Right, right... :laughing: